src/ngraph/frontend/onnx_import/utils/convpool.cpp

```diff
@@ -88,17 +94,16 @@ namespace ngraph
                 return pad_type;
             }
 
-            std::pair<CoordinateDiff, CoordinateDiff> get_pads(const Node& node,
-                                                               const Shape& kernel_shape)
+            std::pair<CoordinateDiff, CoordinateDiff> get_pads(const Node& node, size_t kernel_rank)
```

Comment:
Small nitpick: `kernel_rank` can also be made const.

Reply:
done